experiments/n2v/bsd68/careamics/careamics-bsd68.py

```diff
@@ -63,18 +62,27 @@
 #############################################
 
 # Load test images and ground truth
-test_images = [tifffile.imread(f) for f in sorted(test_path.glob("*.tiff"))]
-gt_images = [tifffile.imread(f) for f in sorted(gt_path.glob("*.tiff"))]
+test_files = sorted(test_path.glob("*.tiff"))
+gt_files = sorted(gt_path.glob("*.tiff"))
 
-# Predict on test images
+# Predict on test images 
 predictions = []
-for test_img in test_images:
+gt_images = []
+
+for test_file, gt_file in zip(test_files, gt_files):
+    test_img = tifffile.imread(test_file)
+    test_img = test_img[np.newaxis, ...] 
+
     pred = careamist.predict(
-        source=test_img,
+        source=test_file, 
```

Comment:
Here you predict on `test_file` which is probably a `path`, but just before you load the image in `test_img`.

It probably makes sense to predict rather on the image you loaded, since you already have it in memory and need it for the metrics computation.